rfc/000-TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+# Proposal Name
+
+- Authors:
```

Comment:
There's no reason for this to be a list.

Reply:
I think having the metadata as itemized list is sensible. How would you do it differently?

Reply:
You added additional metadata fields now, so my comment is obsolete.